test/cholesky/lkjcholesky.jl

```diff
@@ -141,6 +141,7 @@ using FiniteDifferences
             @test insupport(LKJCholesky(40, 2, 'U'), cholesky(rand(LKJ(40, 2))))
             @test insupport(LKJCholesky(40, 2), cholesky(rand(LKJ(40, 2))))
             @test !insupport(LKJCholesky(40, 2), cholesky(rand(LKJ(41, 2))))
+            @test logpdf(LKJCholesky(40, 2), cholesky(rand(LKJ(41, 2)))) == -Inf
```

Comment:
We should make sure that everything is indeed type stable:
```suggestion
            @test @inferred(logpdf(LKJCholesky(40, 2), cholesky(rand(LKJ(41, 2))))) == -Inf
```

In fact, however, most distributions are type stable for `Float64` though, even if implemented incorrectly, so we should add some more tests with other types (`Float32` maybe?) as well.

Reply:
Done. I had to generalise `lkj_onion_loginvconst `.